src/components/AqiGauge/InvalidSensor.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react';
+import {Box, Text, Link} from '@chakra-ui/react';
+import {firestore} from '../../firebase';
+import {formatDate, formatTime} from '../Util/Dates';
+import {useTranslation} from 'react-i18next';
+
+/**
+ * Props for the InvalidSensor component that displays in the AQI Gauge box
+ * when a gray "invalid" sensor is selected
+ * - `purpleAirId` is the PurpleAir ID of the selected sensor
+ */
+interface InvalidSensorProps {
+  purpleAirId: string;
+}
+
+/**
+ * This component is used in the AQI Gauge UI component. When a gray "invalid"
+ * sensor is clicked, instead of showing the AQI Gauge we display a message
+ * that the sensor is down and give the last time of valid reading
+ */
+const InvalidSensor: ({purpleAirId}: InvalidSensorProps) => JSX.Element = ({
+  purpleAirId,
+}: InvalidSensorProps) => {
+  const [lastValidDate, setLastValidDate] = useState('');
+  const [lastValidTime, setLastValidTime] = useState('');
+
+  const {t} = useTranslation('dial');
+
+  if (purpleAirId) {
+    const docRef = firestore.collection('current-reading').doc('sensors');
```

Comment:
To avoid this extra read here, we might consider fetching the data from the `Home` component and then propagating the data using props to the other components.

Reply:
Good point, let's talk about this tomorrow morning, I'm confused about something and would like input

Reply:
Opened an issue for this (because I think it is better handled when you display the reason the sensor is down). #571